aphrodite/common/sampling_params.py

```diff
@@ -143,7 +143,7 @@ def __init__(
         self.max_tokens = max_tokens
         self.logprobs = logprobs
         self.skip_special_tokens = skip_special_tokens
-        self.logits_processors = logits_processors
+        self.logits_processors = logits_processors or []
```

Comment:
Should we add `min_tokens` in SamplingParams?

Reply:
thats my whole last question in OP
right now min_tokens (and ignore_eos) can be implemented just with a logitproc, something that is (so far) only added manually.

I could see sampling_params adding additional logitprocs, but having one parameter automatically set values for another parameters feels off. especially for something like logitprocs, where order potentially matters.

Reply:
Yes ideally min_tokens should be added to params.
Using a logit processor internally is fine IMO, and I don't expect the order to be a real problem(maybe it should be prepended instead of appended but not a big deal).